.github/workflows/dependabot-java-build.yml

```diff
@@ -3,7 +3,7 @@ name: Dependabot build check
 on:
   push:
     branches:
-      - dependabot*
+      - dependabot**
```

Comment:
Why did you add an extra `*`?

Reply:
Extra `*` is used for proper filtering of branches created by Dependabot.
With single `*` branches are not detectable, since default name is `dependabot/...` and slash symbol is not covered by single `*`.

Reply:
Thanks for an explanation